### DISCUSSION TITLE
Does pnpm help against SSD degradation?

### DISCUSSION THREAD

**Discussion by nikoladev** [pnpm/pnpm#3190] [Q&A]
SSDs degrade after a certain amount of time because they can only be written to a certain amount of times. Would pnpm help against this because it doesn't need to write new files all the time because the files are linked from the global store. Or does a link also count as a "write" when it comes to SSD degradation?

Any idea how this could even be tested?

**Comment by zkochan** [comment]
Creating a hard link is also making some changes to the SSD

> Creating an additional hard link has the effect of giving one file multiple names 
https://en.wikipedia.org/wiki/Hard_link

It definitely writes fewer bytes though. Instead of writing a new file, it just writes a new path. I don't know how to test it though. Maybe there are some disk monitoring programs?
